.rubocop_local.yml

```diff
@@ -0,0 +1,3 @@
+GlobalVars:
+  AllowedVariables:
+  - $api_log
```

Comment:
Not sure if you're going to use the same api log or create a new log, but this can be changed later

Reply:
🤔 Probably a different one, but yup, later! Thanks for seeing that, hadn't thought about it really.